### DISCUSSION TITLE
add/modify moose conda environment

### DISCUSSION THREAD

**Discussion by jessecarterMOOSE** [idaholab/moose#16421] [Q&A Getting Started] [answered]
@milljm How should I add extra python packages to my moose conda environment? Should I just add them to the moose conda environment or make a clone and add to the clone? Just wasn't sure if at some point in the future there could be issues if I modify the moose conda environment and try to update it for example. I'm on Linux if it matters.

**Comment by aeslaughter** [comment, marked as answer]
For things I use regularly I just add them to my default 'moose' environment. When I am trying something out I create a new environment.

**Reply by jessecarterMOOSE** [reply]
Sounds good.
